Psyc/Assets/Scripts/FlashingLight.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class FlashingLight : MonoBehaviour
+{
+    public GameObject Light;
+    bool OnOff = true;
```

Comment:
```suggestion
    private bool _isOn= true;
```
Bool gebegint met een werkwoord